ontoconv/ontoflow.py

```diff
@@ -217,10 +219,13 @@ def parse_ontoflow(workflow_data, kb, outdir="."):
     outdir: str
         The directory to save the output files.
         Pipeline and workchain files are saved as yaml.
+    target_kb: Tripper triplestore in which generated output of
```

Comment:
Maybe call it `target_ts` for consistency with the same argument in `generate_ontoflow_pipeline()`